### PR TITLE
FileReader can read from a pipe

### PR DESCRIPTION
**Pull request by stdweird**
partially fixes https://github.com/quattor/CCM/issues/46

### CODE REVIEW

**Comment by stdweird** [comment]
part of https://github.com/quattor/release/issues/81

File: src/main/perl/FileEditor.pm [hunk 1]
```diff
@@ -51,11 +51,19 @@ for C<CAF::FileWriter>
 
 =cut
 
+# FileReader supports reading/editing a file
```
Comment (jrha) [inline src/main/perl/FileEditor.pm]: This is FileEditor?

**Comment by jouvin** [comment]
Sucessfully tested at LAL. Ready to be merged as for me after the comment has been fixed.

**Comment by stdweird** [comment]
@jrha @jouvin comment fixed, commit squashed

**Comment by jrha** [comment]
Thanks!

**Pull request merged**